components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -40,6 +45,8 @@ void Nudging::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
   FieldLayout scalar3d_layout_mid { {COL,LEV}, {m_num_cols, m_num_levs} };
   FieldLayout horiz_wind_layout { {COL,CMP,LEV}, {m_num_cols,2,m_num_levs} };
 
+printf("set grids: %d, %d\n",m_num_cols, m_num_levs);
```

Comment:
Is this left over from debugging?  If we do want to keep this message then I think we should use the atmosphere logger instead of printf.

Reply:
yeah, let me remove it